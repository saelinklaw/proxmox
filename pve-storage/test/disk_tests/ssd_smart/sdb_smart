smartctl 6.6 2016-05-31 r4324 [x86_64-linux-4.4.21-1-pve] (local build)
Copyright (C) 2002-16, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED

SMART Attributes Data Structure revision number: 1
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAGS    VALUE WORST THRESH FAIL RAW_VALUE
  5 Reallocated_Sector_Ct   -O--CK   100   100   000    -    0
  9 Power_On_Hours          -O--CK   100   100   000    -    1259
 12 Power_Cycle_Count       -O--CK   100   100   000    -    191
170 Available_Reservd_Space PO--CK   100   100   010    -    0
171 Program_Fail_Count      -O--CK   100   100   000    -    0
172 Erase_Fail_Count        -O--CK   100   100   000    -    0
174 Unsafe_Shutdown_Count   -O--CK   100   100   000    -    164
175 Power_Loss_Cap_Test     PO--CK   100   100   010    -    5670 (1 343)
183 SATA_Downshift_Count    -O--CK   100   100   000    -    0
184 End-to-End_Error        PO--CK   100   100   090    -    0
187 Reported_Uncorrect      -O--CK   100   100   000    -    0
190 Temperature_Case        -O---K   072   071   000    -    28 (Min/Max 21/30)
192 Unsafe_Shutdown_Count   -O--CK   100   100   000    -    164
194 Temperature_Internal    -O---K   100   100   000    -    28
197 Current_Pending_Sector  -O--C-   100   100   000    -    0
199 CRC_Error_Count         -OSRCK   100   100   000    -    0
225 Host_Writes_32MiB       -O--CK   100   100   000    -    296600
226 Workld_Media_Wear_Indic -O--CK   100   100   000    -    3747
227 Workld_Host_Reads_Perc  -O--CK   100   100   000    -    0
228 Workload_Minutes        -O--CK   100   100   000    -    75111
232 Available_Reservd_Space PO--CK   100   100   010    -    0
233 Media_Wearout_Indicator -O--CK   097   097   000    -    0
234 Thermal_Throttle        -O--CK   100   100   000    -    0/0
241 Host_Writes_32MiB       -O--CK   100   100   000    -    296600
242 Host_Reads_32MiB        -O--CK   100   100   000    -    1207
                            ||||||_ K auto-keep
                            |||||__ C event count
                            ||||___ R error rate
                            |||____ S speed/performance
                            ||_____ O updated online
                            |______ P prefailure warning

