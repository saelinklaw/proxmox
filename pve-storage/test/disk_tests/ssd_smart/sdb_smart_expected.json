{
    "health" : "PASSED",
    "type" : "ata",
    "attributes" : [
	{
	    "name" : "Reallocated_Sector_Ct",
	    "threshold" : 0,
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "raw" : "0",
	    "id" : "  5",
	    "value" : 100
	},
	{
	    "worst" : 100,
	    "name" : "Power_On_Hours",
	    "threshold" : 0,
	    "id" : "  9",
	    "fail" : "-",
	    "raw" : "1259",
	    "flags" : "-O--CK",
	    "value" : 100
	},
	{
	    "value" : 100,
	    "id" : " 12",
	    "raw" : "191",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Power_Cycle_Count"
	},
	{
	    "id" : "170",
	    "raw" : "0",
	    "flags" : "PO--CK",
	    "fail" : "-",
	    "value" : 100,
	    "worst" : 100,
	    "threshold" : 10,
	    "name" : "Available_Reservd_Space"
	},
	{
	    "threshold" : 0,
	    "name" : "Program_Fail_Count",
	    "worst" : 100,
	    "value" : 100,
	    "raw" : "0",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "id" : "171"
	},
	{
	    "worst" : 100,
	    "name" : "Erase_Fail_Count",
	    "threshold" : 0,
	    "id" : "172",
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "raw" : "0",
	    "value" : 100
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Unsafe_Shutdown_Count",
	    "value" : 100,
	    "id" : "174",
	    "raw" : "164",
	    "flags" : "-O--CK",
	    "fail" : "-"
	},
	{
	    "value" : 100,
	    "fail" : "-",
	    "flags" : "PO--CK",
	    "raw" : "5670 (1 343)",
	    "id" : "175",
	    "name" : "Power_Loss_Cap_Test",
	    "threshold" : 10,
	    "worst" : 100
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "SATA_Downshift_Count",
	    "value" : 100,
	    "id" : "183",
	    "raw" : "0",
	    "flags" : "-O--CK",
	    "fail" : "-"
	},
	{
	    "worst" : 100,
	    "name" : "End-to-End_Error",
	    "threshold" : 90,
	    "value" : 100,
	    "id" : "184",
	    "fail" : "-",
	    "raw" : "0",
	    "flags" : "PO--CK"
	},
	{
	    "worst" : 100,
	    "name" : "Reported_Uncorrect",
	    "threshold" : 0,
	    "value" : 100,
	    "id" : "187",
	    "fail" : "-",
	    "raw" : "0",
	    "flags" : "-O--CK"
	},
	{
	    "worst" : 71,
	    "name" : "Temperature_Case",
	    "threshold" : 0,
	    "id" : "190",
	    "fail" : "-",
	    "raw" : "28 (Min/Max 21/30)",
	    "flags" : "-O---K",
	    "value" : 72
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Unsafe_Shutdown_Count",
	    "value" : 100,
	    "id" : "192",
	    "raw" : "164",
	    "flags" : "-O--CK",
	    "fail" : "-"
	},
	{
	    "id" : "194",
	    "fail" : "-",
	    "raw" : "28",
	    "flags" : "-O---K",
	    "value" : 100,
	    "worst" : 100,
	    "name" : "Temperature_Internal",
	    "threshold" : 0
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Current_Pending_Sector",
	    "value" : 100,
	    "id" : "197",
	    "raw" : "0",
	    "flags" : "-O--C-",
	    "fail" : "-"
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "CRC_Error_Count",
	    "value" : 100,
	    "id" : "199",
	    "flags" : "-OSRCK",
	    "raw" : "0",
	    "fail" : "-"
	},
	{
	    "worst" : 100,
	    "name" : "Host_Writes_32MiB",
	    "threshold" : 0,
	    "id" : "225",
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "raw" : "296600",
	    "value" : 100
	},
	{
	    "value" : 100,
	    "id" : "226",
	    "raw" : "3747",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Workld_Media_Wear_Indic"
	},
	{
	    "worst" : 100,
	    "threshold" : 0,
	    "name" : "Workld_Host_Reads_Perc",
	    "id" : "227",
	    "raw" : "0",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "value" : 100
	},
	{
	    "threshold" : 0,
	    "name" : "Workload_Minutes",
	    "worst" : 100,
	    "raw" : "75111",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "id" : "228",
	    "value" : 100
	},
	{
	    "fail" : "-",
	    "raw" : "0",
	    "flags" : "PO--CK",
	    "id" : "232",
	    "value" : 100,
	    "name" : "Available_Reservd_Space",
	    "threshold" : 10,
	    "worst" : 100
	},
	{
	    "value" : 97,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "raw" : "0",
	    "id" : "233",
	    "name" : "Media_Wearout_Indicator",
	    "threshold" : 0,
	    "worst" : 97
	},
	{
	    "value" : 100,
	    "fail" : "-",
	    "raw" : "0/0",
	    "flags" : "-O--CK",
	    "id" : "234",
	    "name" : "Thermal_Throttle",
	    "threshold" : 0,
	    "worst" : 100
	},
	{
	    "worst" : 100,
	    "name" : "Host_Writes_32MiB",
	    "threshold" : 0,
	    "id" : "241",
	    "fail" : "-",
	    "raw" : "296600",
	    "flags" : "-O--CK",
	    "value" : 100
	},
	{
	    "value" : 100,
	    "id" : "242",
	    "fail" : "-",
	    "raw" : "1207",
	    "flags" : "-O--CK",
	    "worst" : 100,
	    "name" : "Host_Reads_32MiB",
	    "threshold" : 0
	}
    ]
}
